docs/settings/subscription.md

```diff
@@ -18,6 +20,14 @@ from: [china-holiday-calender](https://github.com/lanceliao/china-holiday-calend
 
 ### Google Calendar
 
+Go to **'Settings for my calendars'**
+
+Click on the gear icon or right-click on your calendar name to access your calendar settings.
+
+Under the **'Access permissions for events'** section, select **'Make available to public'**. Note that this will make your calendar indexable in searches. Choose **'see only free/busy (hide details)'** to avoid your events being searchable. Read this guide to make sure you're aware of the implications of sharing your calendar: https://support.google.com/calendar/answer/37082?hl=en
+
+Under the **'Integrate calendar'** section, copy the **'Public address in iCal format** and paste that into the url field in Logseq 
```

Comment:
@dario-ds did this work for you?  I got a 404 error when I tried it.

Reply:
Only when I made it public did it work. Before that I was also getting 403 and 404 errors.